src/main/java/core/basesyntax/HasArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface HasArea {
```

Comment:
[Rename](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-verbs-for-classinterface-names).